app/lib/version.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 module Version
-  MAJOR = 1
-  MINOR = 7
-  PATCH = 9
+  MAJOR = 2
```

Comment:
@DanielCouzens this is not the same as the major cadence as the version in the CHANGELOG?

Reply:
@jonrandahl what should the cadence be

It is 1.7.9 in dev

What should I change it to?

Reply:
@DanielCouzens v1.8.0